MFVNeutralino/test/TriggerStudies/eff_electron.py

```diff
@@ -0,0 +1,142 @@
+                             electrons_src = cms.InputTag('slimmedElectrons'),
+                             electron_cut = cms.string('pt > %i && abs(eta) < 2.5' % ele_thresh_offline),
+                             genjets_src = cms.InputTag(''), #'ak4GenJets' if is_mc else ''),
+                             )
+
+process.denht1000 = process.den.clone(require_ht = 1000)
+process.denjet6pt75 = process.den.clone(require_6thjetpt = 75)
+process.denht1000jet6pt75 = process.den.clone(require_ht = 1000, require_6thjetpt = 75)
+process.p = cms.Path(process.weightSeq * process.eletrig * process.updatedJetsSeqMiniAOD * process.BadPFMuonFilterUpdateDz * process.fullPatMetSequence * process.selectedPatJets * process.mfvTriggerFloats * process.den)
+
+#process.dennomu = process.den.clone(require_muon = False)
+#process.dennomuht1000 = process.den.clone(require_muon = False, require_ht = 1000)
+#process.dennomujet6pt75 = process.den.clone(require_muon = False, require_6thjetpt = 75)
+#process.dennomuht1000jet6pt75 = process.den.clone(require_muon = False, require_ht = 1000, require_6thjetpt = 75)
+#process.pnomu = cms.Path(process.weightSeq * process.updatedJetsSeqMiniAOD * process.selectedPatJets * process.mfvTriggerFloats * process.dennomu * process.dennomuht1000 * process.dennomujet6pt75 * process.dennomuht1000jet6pt75)
+
+#for x in '', 'ht1000', 'jet6pt75', 'ht1000jet6pt75', 'nomu', 'nomuht1000', 'nomujet6pt75', 'nomuht1000jet6pt75':
+for x in ['']:
+    num = getattr(process, 'den%s' % x).clone(require_hlt = 20)
+    if 'separate' in weight_l1ecal:
+        num.weight_src = 'jmtWeightMiniAODL1Ecal'
+    setattr(process, 'num%s' % x, num)
+    if 'nomu' in x:
+        process.pnomu *= num
+    else:
+        process.p *= num
+
+import JMTucker.Tools.SimpleTriggerEfficiency_cfi as SimpleTriggerEfficiency
+SimpleTriggerEfficiency.setup_endpath(process)
+
+
+if __name__ == '__main__' and hasattr(sys, 'argv') and 'submit' in sys.argv:
+    import JMTucker.Tools.Samples as Samples
+    from JMTucker.Tools.MetaSubmitter import *
+
+    if year == 2017:
+        samples = Samples.singleelectron_data_samples_2017 + Samples.leptonic_samples_2017 + Samples.met_samples_2017
+        samples += Samples.mfv_splitSUSY_samples_2017
+    elif year == 2018:
+        samples = Samples.egamma_data_samples_2018 + Samples.leptonic_samples_2018 + Samples.met_samples_2018
+    
+    samples = [s for s in samples if s.has_dataset(dataset) and (s.is_mc or not settings.cross)]
+    set_splitting(samples, dataset, 'default', json_path('ana_2017p8.json'), 5)
+
+    ms = MetaSubmitter('TrigEff%s%s' % (version, '_' + settings.cross if settings.cross else ''), dataset=dataset)
+    ms.common.pset_modifier = chain_modifiers(is_mc_modifier, era_modifier, per_sample_pileup_weights_modifier(cross=settings.cross))
+    ms.condor.stageout_files = 'all'
+    ms.submit(samples)
```

Comment:
Could we consolidate the electron trigger stuff into the other eff.py? Since it's only really a few small changes between the two.